tests/p2p/integration_test_helpers.py

```diff
@@ -37,11 +37,13 @@ def __init__(self,
         )
 
     def get_nodes_to_connect(self):
-        nodekey = keys.PrivateKey(decode_hex(
-            "45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"))
-        remoteid = nodekey.public_key.to_hex()
-        yield kademlia.Node(keys.PublicKey(decode_hex(remoteid)),
+        # local-geth's pubkey
+        # pubkey_hex = "3a514176466fa815ed481ffad09110a2d344f6c9b78c1d14afc351c3a51be33d8072e77939dc03ba44790779b7a1025baf3003f6732430e20cd9b76d953391b3"  # noqa: E501
```

Comment:
Is this a local change that slipped into the PR?

Reply:
No, this class (`LocalGethPeerPool`) is a bit of a hack to help me manually test things against a local node. Before it had only the node ID of my geth instance, I just added the node ID for my parity instance now